lib/core/src/Cardano/Byron/Codec/Cbor.hs

```diff
@@ -283,9 +284,7 @@ decodeBlock = do
         1 -> do -- Main Block
             _ <- CBOR.decodeListLenCanonicalOf 3
             h <- decodeMainBlockHeader
-            txs <- decodeMainBlockBody
-            -- _ <- decodeMainExtraData
-            return $ Block h txs
+            Block h <$> decodeMainBlockBody
```

Comment:
I guess this change was to comply with `hlint`, though, we disabled it purposedly here:

```
{-# HLINT ignore decodeBlock "Use <$>" #-}
```

As the do-notation is clearer than the applicative form in this particular case.

Reply:
ah, ok. locally my hlint cried when explicitely invoked, so I made change

Reply:
Maybe check your local version of hlint? I know that the `HLINT` pragma weren't not available in the early versions.

```
$ hlint --version
Hlint v2.2, (C) Neil Mitchell 2006-2019
```